### PR TITLE
Remove HttpResponseStreamWriter

### PR DESCRIPTION
**Pull request by Shane32**
Why does `HttpResponseStreamWriter` exist?  `StreamWriter` has a buffer and we can control its size.  And it can be set to leave the underlying stream open when disposed.  Is there any logical reason to keep the `HttpResponseStreamWriter`?

### CODE REVIEW

**Comment by Shane32** [comment]
If there is a logical reason, I'd like to test the reason to see if it 'holds water'.  Be it speed or whatever.  Just seems like we are replicating existing functionality here.

**Comment by sungam3r** [comment]
I dont't know. This file was added in #809

File: docs2/site/docs/migrations/migration5.md [hunk 1]
```diff
@@ -228,3 +228,4 @@ calls are made to the underlying stream. Only `System.Text.Json` supports asynch
 
 - `InputsConverter` renamed to `InputsJsonConverter`
 - `ExecutionResultContractResolver` renamed to `GraphQLContractResolver`
+- `HttpResponseStreamWriter` removed
```
Comment (sungam3r) [inline docs2/site/docs/migrations/migration5.md:231]:
```suggestion
```

internal

**Comment by Shane32** [comment]
Not sure why but the existing implementation seems better.

```
current

|                 Method |          Code |         Mean |     Error |    StdDev | Ratio |  Gen 0 | Allocated |
|----------------------- |-------------- |-------------:|----------:|----------:|------:|-------:|----------:|
|         NewtonsoftJson | Introspection | 117,416.7 ns | 253.96 ns | 237.56 ns |  1.00 | 0.1221 |   1,512 B |
| NewtonsoftJsonIndented | Introspection | 192,456.1 ns | 487.59 ns | 456.09 ns |  1.64 | 0.2441 |   1,673 B |
|                        |               |              |           |           |       |        |           |
|         NewtonsoftJson |        Middle |   2,675.5 ns |   4.73 ns |   4.42 ns |  1.00 | 0.1755 |   1,104 B |
| NewtonsoftJsonIndented |        Middle |   4,176.9 ns |  11.75 ns |  10.99 ns |  1.56 | 0.1984 |   1,264 B |
|                        |               |              |           |           |       |        |           |
|         NewtonsoftJson |         Small |     724.7 ns |   1.57 ns |   1.39 ns |  1.00 | 0.1411 |     888 B |
| NewtonsoftJsonIndented |         Small |     960.3 ns |   3.76 ns |   3.52 ns |  1.33 | 0.1659 |   1,048 B |



removing HttpResponseStreamWriter 

|                 Method |          Code |         Mean |     Error |    StdDev | Ratio | RatioSD |   Gen 0 |  Gen 1 | Allocated |
|----------------------- |-------------- |-------------:|----------:|----------:|------:|--------:|--------:|-------:|----------:|
|         NewtonsoftJson | Introspection | 121,106.5 ns | 210.20 ns | 175.52 ns |  1.00 |    0.00 | 13.1836 | 2.0752 |  83,480 B |
| NewtonsoftJsonIndented | Introspection | 167,177.4 ns | 323.49 ns | 270.13 ns |  1.38 |    0.00 | 13.1836 | 2.1973 |  83,643 B |
|                        |               |              |           |           |       |         |         |        |           |
|         NewtonsoftJson |        Middle |   4,828.0 ns |  56.05 ns |  52.43 ns |  1.00 |    0.00 | 13.1531 | 2.1896 |  83,072 B |
| NewtonsoftJsonIndented |        Middle |   5,810.3 ns |  32.75 ns |  30.63 ns |  1.20 |    0.02 | 13.1531 | 2.1896 |  83,232 B |
|                        |               |              |           |           |       |         |         |        |           |
|         NewtonsoftJson |         Small |   2,878.0 ns |  43.50 ns |  40.69 ns |  1.00 |    0.00 | 13.1569 | 2.1896 |  82,856 B |
| NewtonsoftJsonIndented |         Small |   3,145.7 ns |  48.68 ns |  45.54 ns |  1.09 |    0.02 | 13.1569 | 2.1896 |  83,016 B |
```

**Comment by sungam3r** [comment]
Internal buffer allocations in StringWriter bits.

**Pull request closed without merging**